src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -524,4 +524,7 @@
   <data name="Use_is_null_check" xml:space="preserve">
     <value>Use 'is null' check</value>
   </data>
+  <data name="AddParenthesisAroundConditionalExpressionInInterpolatedString" xml:space="preserve">
```

Comment:
name is formed by taking the english text and replacing non-alphanumerics with _.  So in this case it would be "Add_parenthesis".

Also, this should be plural.  So "Add parentheses".